CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to
 
 ## [Unreleased]
 
```

Comment:
You also have to update the `src/backend/pyproject.toml` file.

Reply:
Whoops good catch, I already missed it during the last update ..